konfigure-android/src/main/java/nz/co/trademe/konfigure/android/ui/adapter/viewholder/StringConfigViewHolder.kt

```diff
@@ -18,13 +18,13 @@ internal class StringConfigViewHolder(
 
         descriptionTextView.text = model.metadata.description
 
-        currentValueTextView.text = model.value
+        @SuppressLint("SetTextI18n")
+        currentValueTextView.text = "\"${model.value}\""
```

Comment:
I assume this is to distinguish string values from number values?

Reply:
Indeed. It also helps to make empty config values more apparent, as previously they'd show as an empty row.